src/server/views/dashboard/users-delete-confirm.njk

```diff
@@ -0,0 +1,29 @@
+{% extends "./dashboard.njk" %}
+{% import "./macros.njk" as dashboardMacros %}
+{% from "button/macro.njk" import govukButton %}
+
+{% block aside %}
+  <a class="govuk-link govuk-back-link" href="{{ dashboardRoutes.usersEdit.replace(':userEmail', req.params.userEmail) }}">Back</a>
```

Comment:
I think you can just get away with just 
```suggestion
  <a class="govuk-link govuk-back-link" href="{{req.params.userEmail}}">Back</a>
```

Reply:
Didn't quite work. It seems to add the email address at the end of the current url so there were two emails 😥

Reply:
my bad, didn't realise which resource we were in